site/content/deployment/openshift.md

```diff
@@ -10,10 +10,35 @@ to communicate with each other.
 
 ## Prerequisites
 
-The main prerequisite for this kind of deployment is to have an available OpenShift cluster. For a local development, it's pretty simple having such cluster just
-using the OpenShift client tools that can be downloaded from the [OpenShift Origin](https://github.com/openshift/origin/releases) project repository.
+The main prerequisite for this kind of deployment is to have an available OpenShift cluster. For a local development, it's pretty simple having such cluster choosing
+between using :
+
```

Comment:
If I am not mistaken you need the `oc` client in any case, don't you? So I'd rather have the download of the client tools as the prerequisite and then make the distinction between doing a simple `oc cluster up` vs. downloading *minishift* and then `minishift start`.